meddlr/metrics/SSFD.py

```diff
@@ -0,0 +1,205 @@
+"""
+Self-Supervised
+"""
+
+from typing import Sequence
+
+import torch
+from torch import nn
+
+from meddlr.metrics.functional.image import mse
+from meddlr.metrics.metric import Metric
+from meddlr.ops import complex as cplx
+from meddlr.utils import env
+
+SSFD_HUGGINGFACE_URL = "https://huggingface.co/philadamson93/SSFD/resolve/main/model.ckpt"
+
+
+class SSFD(Metric):
```

Comment:
SSFD and LPIPS have a lot of shared structure. Consider writing a `FeatureMetricMixin` which extracts out shared functionality